job_definitions/update_index_alias.yml

```diff
@@ -2,26 +2,30 @@
 - job:
     name: "update-{{ environment }}-index-alias"
     display-name: "Update {{ environment }} index alias"
-    description: Apply a given alias to a given elasticsearch index
+    description: >
+        Create an alias for a given elasticsearch index.
+        If there is an index with that alias already then
+        then index will be given the alias '<alias>-old'.
```

Comment:
We could add a sentence here saying what we actually did: `Look at previous builds for example parameters.`